core/modules/filters/is.js

```diff
@@ -42,24 +42,30 @@ exports.is = function(source,operator,options) {
 	    subops = operator.operand.split("+"),
 		filteredResults = {},
 		results = [];
-	for (var t=0; t<subops.length; t++) {
-		var subop = isFilterOperators[subops[t]];
-		if(subop) {
-			filteredResults[subops[t]] = subop(source,operator.prefix,options);
-		} else {
-			return [$tw.language.getString("Error/IsFilterOperator")];
+	
+	if(subops.length === 1) {
+		var isFilterOperator = isFilterOperators[operator.operand];	
+		if(isFilterOperator) {	
+			return isFilterOperator(source,operator.prefix,options);	
+		} else {	
+			return [$tw.language.getString("Error/IsFilterOperator")];	
+		}
+	} else { 
+		for (var t=0; t<subops.length; t++) {
+			var subop = isFilterOperators[subops[t]];
+			if(subop) {
+				filteredResults[subops[t]] = subop(source,operator.prefix,options);
+			} else {
+				return [$tw.language.getString("Error/IsFilterOperator")];
```

Comment:
duplicate

Reply:
I can't exactly combine that, because in one I need to test every element in a list, and the other a single item.